modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Fields.tsx

```diff
@@ -51,6 +53,12 @@ export default function Fields({
 		setDeletedObjectField,
 	] = useState<ObjectField | null>(null);
 
+	const [load, setLoad] = useState<boolean>(false);
+
+	const [objectFieldTypes, setObjectFieldTypes] = useState<ObjectFieldType[]>(
+		[]
+	);
```

Comment:
Hey @igor-franca, you can create this useState directlly as a map: 
```suggestion
	const [objectFieldTypes, setObjectFieldTypes] = useState<
		Map<string, ObjectFieldType>
	>();
```